### DISCUSSION TITLE
Changing the inactivity timeout for the AFK watcher

### DISCUSSION THREAD

**Discussion by flixmk** [ActivityWatch/activitywatch#723] [Q&A] [answered]
How can I change the time until the watcher counts it as "AFK"? Is this even possible?

**Comment by ErikBjare** [comment, marked as answer]
I'm not entirely sure what you are asking, but there's a configuration option for changing the default 3min timeout used by the AFK watcher to determine if you are AFK or not. I just answered a similar question on the forum: https://forum.activitywatch.net/t/afk-time-details/1771/2

**Reply by zitri3** [reply]
what is aw-watcher-afk-testing? i couldnt find anything in the documentation
![image](https://user-images.githubusercontent.com/77973569/214289462-f896e79c-1474-4568-bb59-f09628c9f2bb.png)

**Reply by ErikBjare** [reply]
@zitri3 The configuration options under the `aw-watcher-afk-testing` header are used when aw-watcher-afk is run in testing mode (with `--testing` passed on the commandline).
